modules/aggregation-layers/src/hexagon-layer/hexagon-aggregator.js

```diff
@@ -39,14 +39,23 @@ export function pointToHexbin({data, radius, getPosition}, viewport) {
   const {iterable, objectInfo} = createIterable(data);
   for (const object of iterable) {
     objectInfo.index++;
-    screenPoints.push(
-      Object.assign(
-        {
-          screenCoord: viewport.projectFlat(getPosition(object, objectInfo))
-        },
-        object
-      )
-    );
+    try {
+      screenPoints.push(
+        Object.assign(
+          {
+            screenCoord: viewport.projectFlat(getPosition(object, objectInfo))
+          },
+          object
+        )
+      );
+    } catch (err) {
+      log.warn(
```

Comment:
- log.warn(...)()
- I'd prefer to skip the row number so that we don't flood the console in case of many invalid positions.
- How useful such a message would be for notebook users? Is there a better place we can surface warnings other than the JavaScript console?

Reply:
Changed appended `()` to `log.warn(...)`, moved the logger outside the loop entirely, and added the boolean check for each input point being finite, rather than the try-catch used in the first version. Also added tests.

I think a wrapper around `log.warn` that plugs into the jupyter widget will be a good way of surfacing a lot of these errors, I'll put out a PR for that sometime soon.

Reply:
> I think a wrapper around log.warn that plugs into the jupyter widget will be a good way of surfacing a lot of these errors, I'll put out a PR for that sometime soon.

`log` is an instance of the probe.gl `Log` class. We could just add a mechanism to let e.g. the JSON API intercept messages so that they could be logged elsewhere than the console, this way users of Log such as deck and luma could remain completely unchanged:

https://github.com/uber-web/probe.gl/blob/master/modules/core/src/lib/log.js#L424

Reply:
@ibgreen made a note of this in the parent ticket and will address in a separate PR